### PR TITLE
tests: refine the flashback raftstore test

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What is changed and how it works?

Issue Number: ref #13303.

What's Changed:

```commit-message
Refine the flashback raftstore test.
```

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

```release-note
None.
```

### CODE REVIEW

File: tests/integrations/raftstore/test_flashback.rs [hunk 1]
```diff
@@ -358,7 +357,7 @@ fn test_flashback_for_apply_snapshot() {
     let local_state = cluster.region_local_state(1, 5);
     assert!(!local_state.get_region().get_is_in_flashback());
 
-    // Add node 3 back.
+    // Add store 3 back.
```
Comment (HuSharp) [inline tests/integrations/raftstore/test_flashback.rs:360]: // Add store 5 back.

**Review by AndreMouche — APPROVED** [review]
LGTM

**Review by Connor1996 — APPROVED** [review]
LGTM

**Pull request merged**